### DISCUSSION TITLE
ONTAP: File System Analytics (FSA) dashboard show only one volume

### DISCUSSION THREAD

**Discussion by XDavidT** [NetApp/harvest#2158] [Q&A] [answered]
When go into "ONTAP: File System Analytics (FSA)" I can see only one volume on the list: https://i.imgur.com/NoyHq0h.png
Just to make sure- I've go to "ONTAP: Volume" and I can see all the volumes.
what can go wrong?

**Comment by rahulguptajss** [comment, marked as answer]
@XDavidT The FSA dashboard currently tracks only the Top 20 volumes with the largest size and analytics enabled per Cluster by default, as specified in the [configuration file](https://github.com/NetApp/harvest/blob/main/conf/rest/9.12.0/volume_analytics.yaml#L10-L12). It seems that you currently have only one volume with analytics enabled for the relevant cluster.

To provide more clarity, we will incorporate this information into the dashboard.

**Reply by XDavidT** [reply]
"analytics enabled for the relevant cluster", is it something that needs to be enabled for each volume?

**Reply by rahulguptajss** [reply]
@XDavidT Before version 9.13, it was necessary to enable FSA for each volume individually. However, in version 9.13, there is now an option to enable FSA by default. For further details, please see below.

https://docs.netapp.com/us-en/ontap/task_nas_file_system_analytics_enable.html
https://docs.netapp.com/us-en/ontap/concept_nas_file_system_analytics_overview.html

![image](https://github.com/NetApp/harvest/assets/25551691/7a1a93bf-6c16-42ac-9dc1-9184c35cb545)


![image](https://github.com/NetApp/harvest/assets/25551691/86c35cb7-e386-4753-b212-86476e0892ff)

**Reply by XDavidT** [reply]
OK thx
